### ISSUE TITLE
It is possible to add render factor as a param in DeepAI

### ISSUE THREAD

**Issue by nahuelhds** [jantic/DeOldify#236]
Hey, @jantic! Congrats for such an excellent tool. I'm using it through DeepAI and I wonder how hard is to add the `render_factor` as a param for API implementation? I could do it my self if you let me access to that code.

What do you say?

**Comment by jantic** [comment]
@nahuelhds Hey! It's possible, for sure, but it's not my call.  I don't have access to the code on that site- it has my name on it but I didn't have anything to do with it other than that they're using DeOldify. I did get asked by them if I thought anything should change about the UI and initially I was going to suggest render_factor, but two things:  

1. It would add an undesired level of complexity for the target audience (it's supposed to be super simple compared to the Colabs).
2. It would add a bunch of extra model hits on the server that weren't happening before per user, greatly adding to the expense of running it (it's free).

Since then I've though of the Colabs as the way to achieve the "power user" mode, and now you can add MyHeritage In Color to that because they actually did add render_factor to their UI.  They did it in a good way- they hid it as an advanced setting.

So in summary- I don't think I'll be advocating for this change at DeepAI.

**Comment by nahuelhds** [comment]
I totally understand and I agree, @jantic. Thanks for taking the time and explain it!
